docs/core/compatibility/core-libraries/6.0/partial-byte-reads-in-streams.md

```diff
@@ -1,62 +1,108 @@
 ---
 title: "Breaking change: Partial and zero-byte reads in DeflateStream, GZipStream, and CryptoStream"
 description: Learn about the .NET 6 breaking change in core .NET libraries where DeflateStream, GZipStream, and CryptoStream handle partial and zero-byte reads differently.
-ms.date: 06/23/2021
+ms.date: 03/18/2024
 ---
 # Partial and zero-byte reads in DeflateStream, GZipStream, and CryptoStream
 
-<xref:System.IO.Compression.DeflateStream>, <xref:System.IO.Compression.GZipStream>, and <xref:System.Security.Cryptography.CryptoStream> diverged from typical <xref:System.IO.Stream.Read%2A?displayProperty=nameWithType> and <xref:System.IO.Stream.ReadAsync%2A?displayProperty=nameWithType> behavior in two ways:
+The `Read()` and `ReadAsync()` methods on <xref:System.IO.Compression.DeflateStream>, <xref:System.IO.Compression.GZipStream>, and <xref:System.Security.Cryptography.CryptoStream> might no longer return as many bytes as were requested.
```

Comment:
`might no longer` sounds odd to me. What about
```suggestion
The `Read()` and `ReadAsync()` methods on <xref:System.IO.Compression.DeflateStream>, <xref:System.IO.Compression.GZipStream>, and <xref:System.Security.Cryptography.CryptoStream> might not return as many bytes as were requested.
```

Reply:
With "might not", it's unclear if it's referring to the old behavior or the new behavior.